frontend/src/views/User.vue

```diff
@@ -1,23 +1,28 @@
 <template>
-    <div class="container is-fluid">
+    <div class="container">
         <div class="section">
-            <b-loading :active="isLoading" :can-cancel="false"></b-loading>
             <h1 class="title">Welcome, {{ userName }}! 🤩</h1>
         </div>
-        <div class="section">
-            <b-tooltip class="is-pulled-right" label="Sync starred repos with github again">
-                <button class="button is-primary has-tooltip" data-tooltip="Tooltip Text">
-                    <b-icon icon="reload"></b-icon>
-                    <span v-on:click="resyncRepos">Resync</span>
-                </button>
-            </b-tooltip>
-        </div>
-        <div class="section">
-            <TagsList></TagsList>
-        </div>
-        <div class="section">
-            <Repos></Repos>
-        </div>
+        <b-loading v-if="isLoading" :active="isLoading" :can-cancel="false" />
+        <template v-else-if="reposToDisplay.length">
```

Comment:
how would reposToDisplay be pulled here? we don't have a hydration call anywhere